jwst/datamodels/tests/test_cube.py

```diff
@@ -0,0 +1,55 @@
+"""Test aspects of CubeModel"""
+import pytest
+
+import numpy as np
+
+from jwst.datamodels import CubeModel
+
+
+# Generate data
+def make_container():
+    """Create the container to test"""
+    size = 5
+    cube = CubeModel((size, size, size))
+    cube.meta.target.proposer_name = 'JWST regresstion test'
+    for idx in range(size):
+        arr = np.zeros((size, size))
+        cube.data[idx] = arr + idx
+        cube.dq[idx] = arr + (idx * 2)
+        cube.err[idx] = arr + (idx * 3)
```

Comment:
Is this part necessary?  The above `cube = CubeModel((size, size, size))` has already initialized the values to zeros.  And it looks like you don't use the values anywhere in the tests.

Reply:
oh yea. was going to use that as an actual data check, but that is probably not necessary. Will remove.